rules/linux/at_command.yml

```diff
@@ -0,0 +1,27 @@
+title: Scheduled Task/Job At
+id: d2d642d7-b393-43fe-bae4-e81ed5915c4b
+status: stable
+description: Detects the use of at/atd
+author: Ömer Günal
+date: 2020/10/06
+references:
+    - https://attack.mitre.org/techniques/T1053/001/
+logsource:
+    product: linux
+detection:
+    keywords:
+        - at|contains:
```

Comment:
the `sigmac` convertor is going to handle `at|contains` as a field name, not as a section name. here is what converter will produce out of your rule:

```
$ ../tools/sigmac -t es-qs -c ../tools/config/elk-linux.yml /tmp/your_rule.yml

( at.keyword:(*\ at\ *) OR 
  atd.keyword:(*\ atd\ *) OR 
  enumeration.keyword:( 
    *which\ atd* OR 
    *which\ at* OR 
    *systemctl\ status\ atd* OR 
    *service\ atd\ status\ *)
)
```

I would do something like that:

```
detection:
    selection:
        CommandLine|contains:
          - ' at '
          - ' atd '
          - 'which atd'
          - 'which at'
          - 'systemctl status atd'
          - 'service atd status '
    condition: selection
```

let me know what do you think about it!